src/pydvl/value/oob/oob.py

```diff
@@ -109,32 +131,28 @@ def compute_data_oob(
     return result
 
 
-def point_wise_accuracy(preds: NDArray, y: NDArray) -> NDArray:
-    r"""Computes point wise accuracy
+def point_wise_accuracy(x1: NDArray[T], x2: NDArray[T]) -> NDArray[T]:
```

Comment:
I think we should use `y_true`, `y_pred` instead of `x1`, `x2` to follow the sklearn convention. It should be more familiar to users.

```suggestion
def point_wise_accuracy(y_true: NDArray[T], y_pred: NDArray[T]) -> NDArray[T]:
```

Reply:
The thing is that this is not tied to any model or prediction. It's just a point-wise metric. As such it doesn't make sense to use y_true and y_pred. But ok, I guess familiarity trumps rigour.